src/core/client/admin/routes/Moderate/Queue/QueueRoute.tsx

```diff
@@ -130,6 +151,7 @@ export const QueueRoute: FunctionComponent<Props> = ({
     subscribeToQueueCommentEntered,
     subscribeToQueueCommentLeft,
```

Comment:
Almost! While this behavior works for `subscribeToQueueCommentEntered `, `subscribeToQueueCommentLeft` must be always on.